src/plugins/data/server/search/es_search/es_search_service.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { coreMock } from '../../../../../core/server/mocks';
+import { EsSearchService } from './es_search_service';
+import { PluginInitializerContext } from '../../../../../core/server';
+import { searchSetupMock } from '../mocks';
+
+describe('ES search strategy service', () => {
+  let service: EsSearchService;
+
+  const mockCoreSetup = coreMock.createSetup();
+  const opaqueId = Symbol();
+  const context: PluginInitializerContext = {
```

Comment:
Not a fan of this, but I didn't see a way to get a mock for this (I might not have been looking in the right places though).